### PR TITLE
Add Wistia Embeds

### PR DESCRIPTION
**Pull request by davidcarlsonberg**
## Type of Change

- **Build Scripts:** Add testing for Wistia embeds.
- **Markdown-It Plugins:** Add support for Wistia embeds.
- **SCSS Styling:** Add styling for Wistia embeds.

## What issue does this relate to?

N/A

### What should this PR do?

Adds Wistia Embed support.

### What are the acceptance criteria?

Wistia embed should properly appear on the index markdown page when running the project. All tests should pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -632,6 +632,33 @@ Set this property to `false` to disable this plugin.
...
+Height and width can optionally be set using `[wistia <id> [height] [width]]`. E.g. `[wistia 7ld71zbvi6 380 560]`.
+The default value for height is 270, and for width is 480.
+
+**Example Markdown input:**
+
+    [wistia 7ld71zbvi6]
+
+**Example HTML output:**
+
+    <iframe src="http://fast.wistia.net/embed/iframe/7ld71zbvi6" class="wistia" height="380" width="560" frameborder="0" allowfullscreen>
```
Comment (MattIPv4) [inline README.md:650]: The height and width here don't match the defaults?

File: rules/embeds/wistia.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+ * Add support for [Wistia](https://fast.wistia.net) embeds in Markdown, as block syntax.
+ *
+ * The basic syntax is `[wistia <id>]`. E.g., `[wistia 7ld71zbvi6]`.
+ * Height and width can optionally be set using `[wistia <id> [height] [width]]`. E.g., `[wistia 7ld71zbvi6 380 560]`.
+ * The default value for height is 270 and for width is 480.
+ *
+ * @example
+ * [wistia 7ld71zbvi6]
+ *
+ * <iframe src="https://fast.wistia.net/embed/iframe/7ld71zbvi6" class="wistia" height="380" width="560" frameborder="0" allowfullscreen>
```
Comment (MattIPv4) [inline rules/embeds/wistia.js:33]: Ditto here on the mismatch

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
:eyes: Can you add an item to the unreleased section of the changelog for this change, please?

File: rules/embeds/wistia.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+/**
+ * @module rules/embeds/wistia
+ */
+
+/**
+ * Add support for [Wistia](https://fast.wistia.net) embeds in Markdown, as block syntax.
+ *
+ * The basic syntax is `[wistia <id>]`. E.g., `[wistia 7ld71zbvi6]`.
+ * Height and width can optionally be set using `[wistia <id> [height] [width]]`. E.g., `[wistia 7ld71zbvi6 380 560]`.
```
[the remark this answers is not in the document]
Comment (davidcarlsonberg) [reply rules/embeds/wistia.js:27]: I've always added a comma and [grammerly suggests](https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/#:~:text=In%20general%2C%20you%20add%20a,associated%20with%20them%2C%20in%20parentheses.) you do so but doesn't specify when its not a list or when its starting a sentence, so I'm happy to go either way I suppose.

[the remark this answers is not in the document]
Comment (MattIPv4) [reply rules/embeds/wistia.js:27]: TIL, learning more oddities about the English language every day

**Review by MattIPv4 — APPROVED** [review]
LGTM

**Pull request merged**